bob-apps/bin/blob_recovery_tool/command.rs

```diff
@@ -201,16 +202,27 @@ pub struct ValidateIndexCommand {
     index_suffix: String,
     delete: bool,
     skip_confirmation: bool,
+    key_size: Option<usize>,
 }
 
 impl ValidateIndexCommand {
     fn run(&self) -> AnyResult<()> {
+        let validate_index_fn = match self.key_size {
+            Some(1) => validate_index::<Key1>,
+            Some(2) => validate_index::<Key2>,
```

Comment:
I think the key sizes `1` and `2` are pointless, so I suggest removing their support (2 bytes key allow to store only 65k records, which is too small for any practical uses of Bob)